l1-contracts/src/mock/StakingAssetHandler.sol

```diff
@@ -176,46 +167,17 @@ contract StakingAssetHandler is IStakingAssetHandler, Ownable {
    * @param _attester - the validator's attester address
    */
   function reenterExitedValidator(address _attester) external override(IStakingAssetHandler) {
```

Comment:
Don't know if there should be a maximum number of times for this, e.g., if you get slashed and exited because you do nothing you could just get back in this way ad infinitum?

Don't seem like a big deal atm because of exit queue and it is something we can address if problem as perephiral